src/content/interop/js-interop/usage.md

```diff
@@ -389,6 +389,45 @@ but can often be elided on and within interop types and on extension members as
 the compiler can tell it is a JS interop type from the representation type and
 on-type.
 
+## Exporting Dart functions and objects to JS
+
+The above sections show how to call JS members from Dart. It's also useful to
+*export* Dart code so that it can be used in JS. To export a Dart function to
+JS, first convert it using [`Function.toJS'], which wraps the Dart function with
```

Comment:
minor: seems from the last edits the second backtick was accidentally changed to a quote, causing this not to render as code and mismatch the link below.

Reply:
Good catch, thanks!